src/App.tsx

```diff
@@ -1,109 +1,43 @@
 import React from 'react';
 
 import './App.scss';
+import { User } from './components/types/Users';
+import { Comments } from './components/types/Comments';
+import { Posts } from './components/types/Posts';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
+import { PostList } from './components/PostList';
 
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+function getUsers(usersId: number): User | undefined {
```

Comment:
```suggestion
function getUsers(usersId: number): User | null {
```

Reply:
use `null `for empty values

Reply:
![image](https://user-images.githubusercontent.com/98764641/205028978-55962b7b-9184-4523-a3d6-b430c04f73dc.png)

Reply:
fix it everywhere